src/libstd/sys/vxworks/fs.rs

```diff
@@ -400,13 +400,27 @@ impl FromInner<c_int> for File {
 
 impl fmt::Debug for File {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        fn get_path(_fd: c_int) -> Option<PathBuf> {
-            // FIXME(#:(): implement this for VxWorks
-            None
+        fn get_path(fd: c_int) -> Option<PathBuf> {
+            let mut buf = vec![0;libc::PATH_MAX as usize];
+            let n = unsafe { libc::ioctl(fd, libc::FIOGETNAME, buf.as_ptr()) };
+            if n == -1 {
+                return None;
+            }
+            let l = buf.iter().position(|&c| c == 0).unwrap();
```

Comment:
I don't know about vxWorks, but is it guaranteed that FIOGETNAME's result is always shorter than PATH_MAX (1024 bytes) including the trailing `\0`?

Reply:
In vxWorks, we are using strlcopy to fill the buffer(arg) like this:
strlcpy ((char *) arg, name, PATH_MAX);